changelog.txt

```diff
@@ -7,6 +7,7 @@
 * Fix - Do not overwrite admin pages when account is disconnected.
 * Update - Set a description when creating payment intents.
 * Add - Use date picker for applicable dispute evidence fields.
+* Fix - Disabled Payment Request Buttons when order has to be split into multiple packages because Payment Requests do not support that use case.
```

Comment:
I don't think we should merge and try to squeeze this into the 2.7.0 release, so we'll need to move the changelog entries to 2.8.0 before we merge.

Reply:
Feel free to merge after rebasing and fixing the changelog version. 👍